starter-code/scripts/models/article.js

```diff
@@ -45,21 +45,28 @@ Article.loadAll = function(inputData) {
 /* This function below will retrieve the data from either a local or remote
  source, process it, then hand off control to the View: */
 Article.fetchAll = function() {
-  if (localStorage.hackerIpsum) {
+  if (localStorage.hackerIpsumObjects) {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    var localStorageData = JSON.parse(localStorage.getItem('hackerIpsumObjects'));
+    Article.loadAll(localStorageData);
+    articleView.renderIndexPage();
```

Comment:
Great work checking to see if the articles are in local storage, parsing them, and passing them to Article.loadAll (to instantiate our article objects and build our articles array), and then finally rendering the index page :)